src/Tribe/Event_Cleaner.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * Class Event_Cleaner
+ */
+class Tribe__Events__Event_Cleaner {
+
+	/**
+	 * @var Tribe__Events__Event_Cleaner
+	 */
+	protected static $instance;
+	private $scheduler;
+
+	/**
+	 * @return Tribe__Events__Event_Cleaner
+	 */
+	public static function instance() {
```

Comment:
We shouldn't create any new classes with the `instance` method, use the Dependency Injection. With `tribe_singleton()` to register the class, and `tribe()` to call it. 

Look at a few examples here: https://github.com/moderntribe/the-events-calendar/blob/master/src/Tribe/Main.php#L392-L445